common-ui/src/main/res/values/design-system-theming.xml

```diff
@@ -277,6 +277,11 @@
         <item name="daxColorSwitchThumbOn">?attr/daxColorAccentBlue</item>
         <item name="daxColorSwitchThumbOff">@color/gray40</item>
 
+        <!-- Slider -->
+        <item name="daxColorSliderTrackInactive">@color/gray40_50</item>
+        <item name="daxColorSliderTrackActive">?attr/daxColorAccentBlue</item>
```

Comment:
`daxColorSliderTrackActive` and `daxColorSliderTrackActive` have the same values in both themes, should we just have one `daxColorSliderActive`?